app.js

```diff
@@ -1,3 +1,4 @@
+const http = require('http');
```

Comment:
im using the package here

Reply:
it's a native node package so you don't need to fetch it elsewhere (if you look at the [source of that package](https://www.npmjs.com/package/http), it looks hella sketchy too - it's kind of a placeholder) - i tested it on my end and it works so if you can confirm the same i can merge the rest of these changes

Reply:
Ohhh okay I see thanks!

Reply:
I'm good to merge this